zotbins/ZotbinsCore
# Removed config generator script

## Description

Removed `genConfig.py` to simplify the development process

## Type of change


## Testing and Verification Instructions

1. Add config.hpp
2. Verify that the project compiles successfully

## Checklist:
